pkg/controller/configmap/configmap_controller.go

```diff
@@ -756,3 +775,29 @@ func secretNotEqual(s1 *corev1.Secret, s2 *corev1.Secret) bool {
 	}
 	return compareKey(s1, s2, "tls.crt") || compareKey(s1, s2, "tls.key") || compareKey(s1, s2, "tls.ca")
 }
+
+func podSecurityPolicySupport(c *rest.Config) bool {
+	version125, _ := version.ParseGeneric("v1.25.0")
+
+	clientset, err := clientset.NewForConfig(c)
+	if err != nil {
+		log.Error(err, "failed to creat clientset")
+		return false
+	}
+
+	serverVersion, err := clientset.Discovery().ServerVersion()
+	if err != nil {
+		log.Error(err, "failed to get running Kubernetes version")
+		return false
+	}
+
+	runningVersion, err := version.ParseGeneric(serverVersion.String())
```

Comment:
What's the difference between serverVersion and runningVersion?

Reply:
Semtically, we can say serverVersion is the same thing with runningVersion, just different representation. 
serverVersion is a var with data type struct, which includes more detailed version info. 
type Info struct {
	Major        string `json:"major"`
	Minor        string `json:"minor"`
	GitVersion   string `json:"gitVersion"`
	GitCommit    string `json:"gitCommit"`
	GitTreeState string `json:"gitTreeState"`
	BuildDate    string `json:"buildDate"`
	GoVersion    string `json:"goVersion"`
	Compiler     string `json:"compiler"`
	Platform     string `json:"platform"`
}

runningVersion is parsed from serverVersion's GitVersion that is a "generic" version string, the format is like
v.1.25 or 1.25. runningVersion is an opqaue representation of a version number

type Version struct {
	components    []uint
	semver        bool
	preRelease    string
	buildMetadata string
}

The reason why we convert serverVersion.string to runningVersion is that we could type Version's provide an unified interface function to handle version info regardless of platform.